### PR TITLE
feature/CORE-2717 fixes for release 2.15.4

### PR DESCRIPTION
**Pull request by codycoggins-virtru**
Update version and add content to README.md

### CODE REVIEW

File: chart/gateway/templates/application.yaml [hunk 1]
```diff
@@ -13,32 +13,33 @@ spec:
   descriptor:
     type: Gateway
     version: "{{ .Values.image.tag }}"
+    description: >-
```
Comment (ehorning) [inline chart/gateway/templates/application.yaml:16]: Are we going to include this generic description in all application charts or is this going to be only in the virtru-public main chart down the road?

**Comment by ehorning** [comment]
Should we add a version file/changelog?

**Review by johndietz — APPROVED** [review]
lgtm

**Comment by codycoggins-virtru** [comment]
> Should we add a version file/changelog?

Yes, definitely.  I'll make this a separate PR to keep the commit history clean.

**Pull request merged**